src/rust/engine/process_executor/src/main.rs

```diff
@@ -11,7 +11,7 @@ use std::collections::BTreeMap;
 ///
 /// It does not perform $PATH lookup or shell expansion.
```

Comment:
Should probably switch to a real CLI arg parsing lib soon, especially since we'll have another CLI tool. [clap](https://crates.io/crates/clap) is supposed to be good.